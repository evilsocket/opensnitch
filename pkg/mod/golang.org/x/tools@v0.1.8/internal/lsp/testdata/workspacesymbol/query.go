package main

// Contains all of the workspace symbol queries.

// -- Fuzzy matching --
//@workspacesymbolfuzzy("rgop")
//@workspacesymbolfuzzy("randoma")
//@workspacesymbolfuzzy("randomb")

// -- Case sensitive --
//@workspacesymbolcasesensitive("main.main")
//@workspacesymbolcasesensitive("p.Message")
//@workspacesymbolcasesensitive("main.myvar")
//@workspacesymbolcasesensitive("main.myType")
//@workspacesymbolcasesensitive("main.myType.Blahblah")
//@workspacesymbolcasesensitive("main.myStruct")
//@workspacesymbolcasesensitive("main.myStruct.myStructField")
//@workspacesymbolcasesensitive("main.myInterface")
//@workspacesymbolcasesensitive("main.myInterface.DoSomeCoolStuff")
//@workspacesymbolcasesensitive("main.embed.myStruct")
//@workspacesymbolcasesensitive("main.embed.nestedStruct.nestedStruct2.int")
//@workspacesymbolcasesensitive("main.embed.nestedInterface.myInterface")
//@workspacesymbolcasesensitive("main.embed.nestedInterface.nestedMethod")
//@workspacesymbolcasesensitive("dunk")
//@workspacesymbolcasesensitive("Dunk")

// -- Standard --
//@workspacesymbol("")
//@workspacesymbol("randomgophervar")
