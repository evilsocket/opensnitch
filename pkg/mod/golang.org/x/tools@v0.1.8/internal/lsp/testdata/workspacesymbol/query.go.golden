-- workspace_symbol-caseinsensitive- --


-- workspace_symbol-caseinsensitive-randomgophervar --
workspacesymbol/a/a.go:3:5-26 RandomGopherVariableA Variable
workspacesymbol/b/b.go:3:5-26 RandomGopherVariableB Variable

-- workspace_symbol-casesensitive-Dunk --
workspacesymbol/main.go:45:6-10 Dunk Function

-- workspace_symbol-casesensitive-dunk --
workspacesymbol/main.go:47:6-10 dunk Function

-- workspace_symbol-casesensitive-main.embed.myStruct --
workspacesymbol/main.go:29:2-10 main.embed.myStruct Field

-- workspace_symbol-casesensitive-main.embed.nestedInterface.myInterface --
workspacesymbol/main.go:40:3-14 main.embed.nestedInterface.myInterface Interface

-- workspace_symbol-casesensitive-main.embed.nestedInterface.nestedMethod --
workspacesymbol/main.go:41:3-15 main.embed.nestedInterface.nestedMethod Method

-- workspace_symbol-casesensitive-main.embed.nestedStruct.nestedStruct2.int --
workspacesymbol/main.go:35:4-7 main.embed.nestedStruct.nestedStruct2.int Field

-- workspace_symbol-casesensitive-main.main --
workspacesymbol/main.go:8:6-10 main.main Function

-- workspace_symbol-casesensitive-main.myInterface --
workspacesymbol/main.go:24:6-17 main.myInterface Interface
workspacesymbol/main.go:25:2-17 main.myInterface.DoSomeCoolStuff Method

-- workspace_symbol-casesensitive-main.myInterface.DoSomeCoolStuff --
workspacesymbol/main.go:25:2-17 main.myInterface.DoSomeCoolStuff Method

-- workspace_symbol-casesensitive-main.myStruct --
workspacesymbol/main.go:20:6-14 main.myStruct Struct
workspacesymbol/main.go:21:2-15 main.myStruct.myStructField Field

-- workspace_symbol-casesensitive-main.myStruct.myStructField --
workspacesymbol/main.go:21:2-15 main.myStruct.myStructField Field

-- workspace_symbol-casesensitive-main.myType --
workspacesymbol/main.go:14:6-12 main.myType Class
workspacesymbol/main.go:18:18-26 main.myType.Blahblah Method

-- workspace_symbol-casesensitive-main.myType.Blahblah --
workspacesymbol/main.go:18:18-26 main.myType.Blahblah Method

-- workspace_symbol-casesensitive-main.myvar --
workspacesymbol/main.go:12:5-10 main.myvar Variable

-- workspace_symbol-casesensitive-p.Message --
workspacesymbol/p/p.go:3:7-14 p.Message Constant

-- workspace_symbol-fuzzy-randoma --
workspacesymbol/a/a.go:3:5-26 RandomGopherVariableA Variable
workspacesymbol/a/a.go:5:7-28 RandomGopherConstantA Constant
workspacesymbol/a/a.go:8:2-24 randomgopherinvariable Constant
workspacesymbol/a/a_test.go:3:5-30 RandomGopherTestVariableA Variable
workspacesymbol/a/a_x_test.go:3:5-31 RandomGopherXTestVariableA Variable
workspacesymbol/b/b.go:3:5-26 RandomGopherVariableB Variable
workspacesymbol/b/b.go:6:2-5 RandomGopherStructB.Bar Field

-- workspace_symbol-fuzzy-randomb --
workspacesymbol/a/a.go:3:5-26 RandomGopherVariableA Variable
workspacesymbol/a/a.go:8:2-24 randomgopherinvariable Constant
workspacesymbol/a/a_test.go:3:5-30 RandomGopherTestVariableA Variable
workspacesymbol/a/a_x_test.go:3:5-31 RandomGopherXTestVariableA Variable
workspacesymbol/b/b.go:3:5-26 RandomGopherVariableB Variable
workspacesymbol/b/b.go:5:6-25 RandomGopherStructB Struct
workspacesymbol/b/b.go:6:2-5 RandomGopherStructB.Bar Field

-- workspace_symbol-fuzzy-rgop --
workspacesymbol/a/a.go:3:5-26 RandomGopherVariableA Variable
workspacesymbol/a/a.go:5:7-28 RandomGopherConstantA Constant
workspacesymbol/a/a.go:8:2-24 randomgopherinvariable Constant
workspacesymbol/a/a_test.go:3:5-30 RandomGopherTestVariableA Variable
workspacesymbol/a/a_x_test.go:3:5-31 RandomGopherXTestVariableA Variable
workspacesymbol/b/b.go:3:5-26 RandomGopherVariableB Variable
workspacesymbol/b/b.go:5:6-25 RandomGopherStructB Struct
workspacesymbol/b/b.go:6:2-5 RandomGopherStructB.Bar Field

